src/Exception/BaseGroupException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace AppBundle\Exception;
+
+use Ramsey\Uuid\UuidInterface;
+
+class BaseGroupException extends \RuntimeException
+{
+    private $groupUuid;
+
+    public function __construct(UuidInterface $groupUuid, $message = '', \Exception $previous = null)
+    {
+        parent::__construct($message, 0, $previous);
+
+        $this->groupUuid = $groupUuid;
+    }
+
+    public function getCommitteeUuid(): UuidInterface
```

Comment:
Does it make sense to have two methods that return the same value. The name is is misleading though.